make DBMS_CRYPTO optional

lee-lindley opened the pull request:
Resolves #5 

Add a sqlplus define to install.sql and use sqlplus syntax to conditionally compile xutl_offcrypto with a plsql_ccflags setting to remove references to encryption from ExcelGen package.

The added complexity of install.sql may not suit. If not, I would be happy to create "install_nocrypto.sql" instead. That would eliminate dependence on sqlplus tricks for conditional compilation. The plsql_ccflags conditional compilation would remain.

mbleron commented:
Sorry, late reply.

Indeed I would prefer a dedicated `install_nocrypto` script. Thanks.

lee-lindley commented:
As you asked, reverted install.sql and added install_nocrypto.sql. Updated README.md with reference to it and added an entry in ChangeLog section.
I ran both install.sql and install_nocrypto.sql. Both compile fine. I've been ExcelGen without DBMS_CRYPTO in a production environment for serveral months with no issues.

PS
I wish there was a less convoluted way to do conditional compilation in sqlplus, but that trick with sqlplus COLUMN variable is how Oracle does it in release code. I agree it is too weird to throw around lightly and concur this is a less confusing way to do it.
